web-components/src/components/icons/AvailableCreditsIconAlt.tsx

```diff
@@ -21,8 +21,8 @@ export default function AvailableCreditsIcon({
         xmlns="http://www.w3.org/2000/svg"
       >
         <path
-          fill-rule="evenodd"
-          clip-rule="evenodd"
+          fillRule="evenodd"
+          clipRule="evenodd"
```

Comment:
change to valid React properties

Reply:
thanks, these are easy to miss sometimes